src/App.tsx

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import './App.css';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Signup from './components/Signup';
+import Signin from './components/Signin';
+import Menubar from './components/Menubar';
 
 function App() {
     return (
         <BrowserRouter>
             <Routes>
                 <Route path='/signup' element={<Signup />} />
+                <Route path='/signin' element={<Signin />} />
+                <Route path='/menubar' element={<Menubar />} />
```

Comment:
I dont think we'll want the Menubar to be its own route. We would want this to be a wrapper of various pages.

But if the intention is to have this temporary so we can preview it then im okay with keeping this in the mean time